test/client_integration/gdas-marine-templates.yaml

```
@@ -36,6 +36,7 @@ number_of_outer_loops: 1
 # Model things
 # ------------
 marine_soca_files_path: DATA/soca
+marine_vt_levels: 25
```

```suggestion
marine_vt_levels: 25
marine_stddev_time: '2024-01-31T21:00:00Z'
```

This will get the tests passing.